src/SDKs/DataLake.Store/Management.DataLake.Store/Microsoft.Azure.Management.DataLake.Store.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataLake.Store</PackageId>
 		<Description>Provides Data Lake Store account and filesystem management capabilities for Microsoft Azure.</Description>
-		<Version>2.4.0-preview</Version>
+		<Version>2.5.0-preview</Version>
```

Comment:
@lewu-msft Looks like you are only making additive changes here, theoretically you could release `2.4.1-preview` version, however I leave it your discretion